frontend/src/translations/en.js

```
@@ -0,0 +1,178 @@
+  dueToday: 'Due Today',
+  dueTomorrow: 'Due Tomorrow',
+  upcoming: 'Upcoming',
+  completed: 'Completed',
+  pending: 'Pending',
+  all: 'All',
+  filterBy: 'Filter by',
+  sortBy: 'Sort by',
+  search: 'Search',
+  duration: 'Duration',
+  minutes: 'minutes',
+  hours: 'hours',
```

Biome (2.4.7)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)